pkg/Cpanel/Security/Advisor/Assessors/Processes.pm

```diff
@@ -105,10 +107,16 @@ sub _check_for_outdated_processes {
 
     if (@services) {
         my $restart_cmd = 'systemctl restart ' . join( q{ }, @services );
-        if ( !Cpanel::Sys::OS::Check::has_systemd() ) {
+        my $systemd     =
+            $has_modern_cpanel_os
+            ? Cpanel::OS::is_systemd()
+            : Cpanel::OS::Check::has_systemd();
```

Comment:
@xsawyerx Noticed the bug here too late - `Cpanel::OS::Check::has_systemd();` should be `Cpanel::Sys::OS::Check::has_systemd()`

Reply:
Opened CPANEL-38113 to address this bug.

Reply:
Marked as resolved. (Sorry!)